contrib/instrumentation.sh

```diff
@@ -3,3 +3,7 @@
 adb logcat -c
 adb logcat *:E -v color &
 ./gradlew jacocoInstrumentationTestReport
+adb remount
```

Comment:
Screenshots are overwriting each other for this workflow run as they all have the same name so we can't get a specific image for a failure. If we can pass the api level to the bash script/uploader so the filenames are eg `screencap-21` then we can assess if we are getting useful screenshots. If these screenshots are not useful then me may have to move the screenshot taking internal to the code much like `applyViewHeirarchyPrinting`

Also guessing from the error messages [this](https://stackoverflow.com/questions/40488317/adb-error-remote-object-data-data-com-me-myproject-does-not-exist) might work for API 21/22?

Reply:
@macgills I wasn't able to get the screenshots in API 21 and 22. I tried what that link had, but I'm already pulling the screenshots from the `sdcard` directory.

Reply:
I don't think I saw a commit with this line
`run-as com.yourPackageName`

and when you were trying to use `/mnt/sdcard/` you were copying to `/sdcard/` first which is the problematic directory.

This is what I remember seeing so I could be inaccurate